src/logger.ts

```diff
@@ -128,14 +128,26 @@ interface PrefixedLogger extends loglevel.Logger, LoggerWithLogMethod {
  *
  * @param prefix Prefix to add to each logged line. If undefined, no prefix will be added.
  */
-function getPrefixedLogger(prefix?: string): LoggerWithLogMethod {
+function getPrefixedLogger(prefix?: string): PrefixedLogger {
     const loggerName = DEFAULT_NAMESPACE + (prefix === undefined ? "" : `-${prefix}`);
     const prefixLogger = loglevel.getLogger(loggerName) as PrefixedLogger;
-
+    // This is a save cast since export const logger is constructed with getPrefixedLogger
+    // It is downcastet to `Logger` to minimize the public/exported api.
```

Comment:
This comment looks like it's out of place - should it go on the `const rootLogger = logger as PrefixedLogger;` line?